watson/watson.py

```diff
@@ -425,3 +433,10 @@ def merge_report(self, frames_with_conflict):
                 merging.append(conflict_frame)
 
         return conflicting, merging
+
+    def _default_tags(self, project):
+        default_tags_raw = self.config.get('default_tags', project)
+        default_tags = []
+        if default_tags_raw is not None:
+            default_tags = shlex.split(default_tags_raw)
+        return default_tags
```

Comment:
Ok, this took quite some time, but #114 is now merged. Please update our PR, rebase it on the current master and squash the commits. Then it can be merged.